document STACK_DRIVER_LOGGING_ENABLED (#714)
harness/developer-hub: docs/platform/2_Delegates/delegate-reference/delegate-environment-variables.md

  value: "false" 
```

### STACK_DRIVER_LOGGING_ENABLED

> By default delegates send logs to harness for debugging and support. Set this to false if you want to disable sending logs to us. 

Reviewer: ```suggestion
Delegates send logs to Harness by default. Harness uses these logs for debugging and support. To disable logs from being sent to Harness, set this value to `false`. 
```
